script/travis

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
That's fair then, after I finish up the ObjectSpace removal I'll add a usage info.